en/micro-integrator/docs/setup/deployment/migrating-from-ei-7.0.0.md

```diff
@@ -0,0 +1,51 @@
+# Migrating from WSO2 EI 7.0.0 to WSO2 EI 7.1.0
+
+### Set up the migration
+
+-	Make a backup of the existing database used by the current EI 7.0.0 deployment. This backup is necessary in case the migration causes any issues in the existing database.
```

Comment:
We didn't support databases in EI 7.0.0 did we? I think we can remove this section.
@Nirothipan - could you also confirm?

Reply:
Yes +1 . I guess this has been taken from migrating from 6xx doc.

Reply:
Removed.